node-commons/src/main/java/org/apache/iotdb/commons/service/metric/enums/Metric.java

```diff
@@ -86,6 +86,7 @@ public enum Metric {
   DRIVER_SCHEDULER,
   PERFORMANCE_OVERVIEW,
   PERFORMANCE_OVERVIEW_DETAIL,
+  PERFORMANCE_OVERVIEW_SCHEDULE_OVERVIEW,
   PERFORMANCE_OVERVIEW_SCHEDULE_DETAIL,
```

Comment:
do we need a `PERFORMANCE_OVERVIEW_STATEMECHINE_DETAIL` for those operations in dataRegion?